modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/client_http_invoker.ftl

```diff
@@ -356,14 +356,16 @@ public class HttpInvoker {
 
 		byte[] bytes = new byte[8192];
```

Comment:
I think this variable should be defined inside the if clause, not outside

Reply:
Thanks for catching that!